src/jit/jiteh.cpp

```diff
@@ -2426,6 +2426,11 @@ bool Compiler::fgNormalizeEHCase2()
                                 // this once per dup.
                                 fgReplaceJumpTarget(predBlock, newTryStart, insertBeforeBlk);
 
+                                // Need to adjust ref counts here since we're retargeting edges.
+                                newTryStart->bbRefs++;
+                                noway_assert(insertBeforeBlk->countOfInEdges() > 0);
```

Comment:
EH normalization is always done, and runs early enough that nothing upstream that is disabled by minopts has yet run, so I suppose I could change this.